### PR TITLE
Spruce up sbt organization.

### PR DESCRIPTION
**Pull request by olix0r**
Namerd.dcosBootstrap should be part of the Namerd.all aggregate.

### CODE REVIEW

File: project/LinkerdBuild.scala [hunk 1]
```diff
@@ -529,6 +529,18 @@ object LinkerdBuild extends Base {
...
   // Unified documentation via the sbt-unidoc plugin
   val all = project("all", file("."))
-    .aggregate(k8s, consul, marathon, Linkerd.all, Namerd.all, Namerd.dcosBootstrap, Router.all, Namer.all, Interpreter.all, configCore, admin, testUtil)
+    .aggregate(
+      admin,
+      configCore,
+      consul,
+      k8s,
+      testUtil
+      marathon,
```
Comment (klingerf) [inline project/LinkerdBuild.scala]: Is this intended to be alphabetical?  In that case I'd put testUtil after marathon.

Comment (olix0r) [inline project/LinkerdBuild.scala]: doh! good catch

**Comment by klingerf** [comment]
⭐ 💥

**Comment by siggy** [comment]
⭐ 👍

**Comment by rmars** [comment]
:star: 🍃

**Comment by stevej** [comment]
❇️ 👍

**Pull request merged**